policyengine/utils/charts.py

```diff
@@ -32,6 +32,7 @@ def add_fonts():
 WHITE = "#FFFFFF"
 TEAL_98 = "#F7FDFC"
 BLACK = "#000000"
+LIGHT_LIGHT_GRAY = "#F4F4F4"
 
 BLUE_COLOUR_SCALE = [
```

Comment:
```suggestion
BLUE_COLOR_SCALE = [
```
__nitpick:__ Standardize to American spelling conventions

Please don't hate me for this one

Reply:
Done!